tests/unit/services/cookies-test.js

```diff
@@ -187,6 +187,13 @@ describe('CookiesService', function() {
         expect(document.cookie).to.include(`${COOKIE_NAME}=${encodeURIComponent(value)}`);
       });
 
+      it("doesn't encode the value when encodeValue is false", function() {
+        let value = '!"§$%&/()=?"';
+        this.subject().write(COOKIE_NAME, value, { encodeValue: false });
+
+        expect(document.cookie).to.include(`${COOKIE_NAME}=${value}`);
+      });
```

Comment:
A similar test would have to be added to the section that tests correct behavior in FastBoot below.